core/helpers/EEH_Schema.helper.php

```diff
@@ -48,6 +49,18 @@ public static function add_json_linked_data_for_event(EE_Event $event)
         $template_args['event_start'] = $primary_datetime->start_date(DateTime::ATOM);
         $template_args['event_end'] = $primary_datetime->end_date(DateTime::ATOM);
         unset($primary_datetime);
+        switch ($event->status()) {
+            case 'cancelled':
+                $event_status = 'EventCancelled';
+                break;
+            case 'postponed':
+                $event_status = 'EventPostponed';
+                break;
+            default:
+                $event_status = 'EventScheduled';
+                break;
+        }
```

Comment:
It's best to use constants for the values returned for status because it ensures they are valid
and you don't _need_ a break for `default` if it is the last case in the switch
(doesn't cause any problems if left in however, but can be removed)

```suggestion
        switch ($event->status()) {
            case EEM_Event::cancelled:
                $event_status = 'EventCancelled';
                break;
            case EEM_Event::postponed:
                $event_status = 'EventPostponed';
                break;
            default:
                $event_status = 'EventScheduled';
        }
```